src/components/votes/votesTab.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { translate } from 'react-i18next';
+import BoxV2 from '../boxV2';
+import AccountVisual from '../accountVisual';
+import VotesTableHeader from './votesTableHeader';
+import TableRow from '../toolbox/table/tableRow';
+import SpinnerV2 from '../spinnerV2/spinnerV2';
+import { InputV2 } from '../toolbox/inputsV2';
+import styles from './votesTab.css';
+
+class VotesTab extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      showing: 30,
+      filterValue: '',
+    };
+
+    this.onShowMore = this.onShowMore.bind(this);
+    this.handleFilter = this.handleFilter.bind(this);
+  }
+
+  onShowMore() {
+    const max = this.props.votes.length;
+    const showing = (this.state.showing + 30 > max) ? max : this.state.showing + 30;
+    this.setState({ showing });
+  }
+
+  handleFilter({ target }) {
+    this.setState({
+      filterValue: target.value,
+    });
+  }
+
+  render() {
+    const { t, votes, loading } = this.props;
+    const { filterValue } = this.state;
+    const filteredVotes = votes ? votes.filter(vote => RegExp(filterValue, 'i').test(vote.username)) : [];
+    const canLoadMore = filteredVotes.length > this.state.showing;
+    const isLoading = loading.length > 0;
+
+    return (
+      <BoxV2 className={`${styles.wrapper}`}>
+        <header>
+          <h1>{t('Voted delegates')}</h1>
+          <div className={`${styles.filterHolder}`}>
+            <InputV2
+              disabled={votes && votes.length === 0}
+              name={'filter'}
+              value={filterValue}
+              placeholder={t('Filter by name')}
+              onChange={this.handleFilter} />
+          </div>
+        </header>
+        <main className={`${styles.results} ${canLoadMore ? styles.hasMore : ''} ${isLoading ? styles.isLoading : ''}`}>
+          <VotesTableHeader />
+          {
+            isLoading ? (
+              <div className={styles.loadingOverlay}>
+                <SpinnerV2 className={`${styles.loadingSpinner}`} />
+              </div>
+            ) : null
+          }
+          {filteredVotes.length > 0
```

Comment:
I think in here too.
you can just use filteredVotes.length.
By the way, maybe you can try to use the same way in all the files when open/close brackets for dont affect the indentation.
in line 60 you open brackets and then use the next line to code but in here you open brackets and the code next to the brackets, so it is not a real problem is just that is easier to read when you do the same and not change it very often and the indentation is better to follow. Same in lines 97 and 103
(really minor details nothing related to the code) 👍🏻